src/wallet/rpcwallet.cpp

```diff
@@ -3501,6 +3192,10 @@ UniValue spendmany(const UniValue& params, bool fHelp) {
 
     LOCK2(cs_main, pwalletMain->cs_wallet);
 
+    // Only account "" have sigma coins.
```

Comment:
Disagree. Should remove `account` option altogether. Otherwise it's a useless parameter

Reply:
The reason we make this RPC to use the same format as `sendmany` because we want to make it the same except just RPC name. That why we still keep that parameter even we did not use it. @reubenyap do you have any idea?

Reply:
@ultimaweapon Why must it be the same? By that logic, users will think that they HAVE the option of using a different account. At the very least, the error message should be changed to something along the lines of `Account is not usable with Sigma coins` or so

Reply:
The benefit we have to make it the same is:

1. User just need to change the only RPC name to switch between normal send and private send.
2. It will easier for scripting too since it just need to change only RPC name to switch between normal send and private send.

Usually the RPC is for advance user and programming so I think it make sense to go with this way. The account name for `sendmany` also already deprecated and it accept only empty string too. The reason it still there instead of remove is because backward compatibility. The solution that Bitcoin use is they provide the description about this parameter that it is already deprecated and accept only empty string.

I believe the reason we use insufficient fund error because we want to mimic the behavior of `sendmany`.

Reply:
Ok, from the perspective of ease-of-use when scripting, I agree with you. People using it manually (ie. most likely programmers) will figure it out I guess.

Reply:
thanks Putta for explanation.